source/client/output_collector_impl.h

```diff
@@ -50,5 +50,11 @@ class YamlOutputCollectorImpl : public OutputCollectorImpl {
   std::string toString() const override;
 };
 
+class DottedStringOutputCollectorImpl : public OutputCollectorImpl {
```

Comment:
Can you link to where this format is specified? i.e. is there an RFC or something?

Reply:
I'm actually not sure if there's a formal spec. @jplevyak would you know?